src/Silex/Provider/RoutingServiceProvider.php

```diff
@@ -76,7 +76,12 @@ public function register(Container $app)
                 return $app['request_matcher'];
             });
 
-            return new RouterListener($urlMatcher, $app['request_stack'], $app['request_context'], $app['logger']);
+            return new RouterListener(
+                $urlMatcher,
+                $app['request_stack'],
+                $app['request_context'],
+                $app['logger.request']
```

Comment:
I'd recommend to have a different logger - `logger.router` for the route matching. Not only, because this is not about the request, but also, because route matches are being logged on the `INFO` level in contrast to other loggers logging on the `DEBUG` level and users might want to do something different only with `logger.router` like changing the log level threshold.